src/patternfly/components/Nav/examples/index.js

```diff
@@ -30,7 +30,7 @@ import docs from '../docs/code.md';
 
 export const Docs = docs;
 
-export default (props) => {
+export default props => {
```

Comment:
did you mean to commit this?

@redallen do you know if this is necessary? Looks like every component/layout has them except for

```
marvin:patternfly-next cmichael$ grep -ri 'export default' src/ | grep props | grep -v '(props)'
src//patternfly/components/Alert/examples/index.js:export default props => {
src//patternfly/components/AppLauncher/examples/index.js:export default props => {
src//patternfly/components/NotificationBadge/examples/index.js:export default props => {
src//patternfly/components/AlertGroup/examples/index.js:export default props => {
```